examples/platforms/efr32mg12/system.c

```diff
@@ -59,9 +59,8 @@
 #include "fem-control.h"
 #endif
 
-#define USE_EFR32_LOG                                                                   \
-    ((OPENTHREAD_CONFIG_LOG_OUTPUT == OPENTHREAD_CONFIG_LOG_OUTPUT_PLATFORM_DEFINED) || \
-     (OPENTHREAD_CONFIG_LOG_OUTPUT == OPENTHREAD_CONFIG_LOG_OUTPUT_NCP_SPINEL))
+#define USE_EFR32_LOG \
+    ((OPENTHREAD_CONFIG_LOG_OUTPUT == OPENTHREAD_CONFIG_LOG_OUTPUT_PLATFORM_DEFINED)
```

Comment:
@bukepo there's a typo in the about change.  There's an extra '(' at the beginning which results in this compilation error:

openthread/examples/../examples/platforms/efr32mg12/system.c:63:5: error: missing ')' in expression
     ((OPENTHREAD_CONFIG_LOG_OUTPUT == OPENTHREAD_CONFIG_LOG_OUTPUT_PLATFORM_DEFINED)
     ^
openthread/examples/../examples/platforms/efr32mg12/system.c: In function 'otSysDeinit':
openthread/examples/../examples/platforms/efr32mg12/system.c:63:5: error: missing ')' in expression

Reply:
I created https://github.com/openthread/openthread/pull/5135 to address the issue.